Tests/SentrySwiftUITests/PreviewRedactOptionsTests.swift

```diff
@@ -0,0 +1,140 @@
+@testable import SentrySwiftUI
+import XCTest
+
+class PreviewRedactOptionsTests: XCTestCase {
+    func testInit_whenWithoutArguments_shouldUseDefaultOptions() {
+        // -- Act --
+        let options = PreviewRedactOptions()
+
+        // -- Assert --
+        XCTAssertTrue(options.maskAllText)
+        XCTAssertTrue(options.maskAllImages)
+        XCTAssertTrue(options.maskedViewClasses.isEmpty)
+        XCTAssertTrue(options.unmaskedViewClasses.isEmpty)
+        XCTAssertFalse(options.enableViewRendererV2)
+    }
+
+    func testInit_withAllArguments_shouldSetOptions() {
+        // -- Arrange --
+        let options = PreviewRedactOptions(
+            maskAllText: false,
+            maskAllImages: false,
+            maskedViewClasses: [UIView.self],
+            unmaskedViewClasses: [UIButton.self],
+            enableViewRendererV2: true
+        )
+        // -- Assert --
+        XCTAssertFalse(options.maskAllText)
+        XCTAssertFalse(options.maskAllImages)
+        XCTAssertEqual(options.maskedViewClasses.count, 1)
+        for maskedViewClass in options.maskedViewClasses {
+            XCTAssertEqual(String(describing: maskedViewClass), String(describing: UIView.self))
+        }
```

Comment:
`l`: Maybe we can use `assertArrayEquals` or some other helper method to avoid having to write this assertion code multiple times.